src/modules/settings/Settings.ts

```diff
@@ -25,6 +25,11 @@ export default class Settings {
         return this.list.find((setting) => setting.name === name) || null;
     }
 
+    static isContainedInMultiSetting(name: string, value: any) {
+        const setting = Settings.getSetting(name);
+        return setting !== undefined && Array.isArray(setting.value) && setting.value.includes(value);
+    }
+
```

Comment:
This feels like it should just be an override of `isSelected` in the MultiSetting class

Reply:
Changed to be an override of isSelected